src/plugin.js

```diff
@@ -2,6 +2,10 @@ import videojs from 'video.js';
 import window from 'global/window';
 import document from 'global/document';
 
+// The logic below is used to check if flash is disabled or not installed
+// in IE browser and show the appropriate message to the user.
+const flashObj = videojs.getComponent('Flash');
```

Comment:
The comment for this seems to be inaccurate now. It could probably just be removed.

And I think this should be named consistently with other components/classes with a capital first letter (i.e. `Flash`).